src/frontend/github-oauth.ts

```diff
@@ -58,13 +55,14 @@ export default (opts: OAuthOptions): GithubOAuth => {
     next();
   }
 
-  async function callback(req, res, next): Promise<void> {
+  async function callback(req: Request, res: Response, next: NextFunction): Promise<void> {
+    const requestStart = Date.now();
     const { query } = url.parse(req.url, true);
     const code = query.code as string;
     const state = query.state as string;
 
     // Take save redirect url and delete it from session
-    const redirectUrl = req.session[state];
+    const redirectUrl = req.session[state] || '';
```

Comment:
Did this give you issues?  It should always be there when the callback gets called...

Reply:
See above comment.